api/allIncidents/incidentsRouter.js

```diff
@@ -23,15 +23,12 @@ const {
  *      500:
  *        description: Server response error
  */
-
 router.get('/', validateAndSanitizeIncidentQueries, (req, res, next) => {
```

Comment:
validateAndSani.. middleware is not being used properly here. We'll need a new endpoint/model function if we want to accept query's to the database.

Reply:
Agreed, if that's deprecated middleware it can be deleted, if anything the commit id can be saved in the trello for future use.

Reply:
I agree with deleting whatever is not needed should be deleted